postSkybill.php

```diff
@@ -11,7 +11,7 @@
 */
 
 if (!defined('_MYDIR_')) define('_MYDIR_', dirname(__FILE__));
-require('../../config/config.inc.php');
+require($_SERVER['DOCUMENT_ROOT'].'/config/config.inc.php');
```

Comment:
This won't work in most deployments cases !
Can you resubmit the PR using `_PS_ROOT_DIR_` instead ?